autoinstrumentation/python/Dockerfile

```diff
@@ -17,8 +18,18 @@ ADD requirements.txt .
 
 RUN mkdir workspace && pip install --target workspace -r requirements.txt
 
+FROM python:3.11-alpine AS build-musl
+
+WORKDIR /operator-build
+
+ADD requirements.txt .
+
+RUN apk add --update --no-cache gcc musl-dev linux-headers
+RUN mkdir workspace && pip install --target workspace -r requirements.txt
+
 FROM busybox
 
-COPY --from=build /operator-build/workspace /autoinstrumentation
+COPY --from=build /operator-build/workspace /autoinstrumentation/linux-x64
```

Comment:
I am not sure how in details looks release process for docker images, but if it release new version just after merge to main it can be breaking change for currently released version.

If the new image will be applicable only to the new operator release - then it LGTM.

Reply:
A very good concern! I will check